src/components/common/Input/Input.tsx

```diff
@@ -28,6 +28,7 @@ const Input = ({
       <label htmlFor={name}>{label} :</label>
       <div className="input-field">
         <input
+          id={name}
```

Comment:
We took the id field out because it was causing a DOM error, let's not add it again until we work around the issue, it's not needed.